src/cbapi/response/models/ingress_filter.yaml

```diff
@@ -0,0 +1,120 @@
+type: object
+required:
+- id
+- name
+- description
+properties:
+  id:
+    type: string
+    description: "Unique filter identifier, in order to allow future updates. Can be any string, but should contain only alfanumeric digits and dashes (e.g. uuid). This is read-only field and honored only during initial POST."
+  create_timestamp:
+    type: string
+    format: "date-time"
+    description: "Date of creation of this filter. This is read-only field."
+  modify_timestamp:
+    type: string
+    format: "date-time"
+    description: "Date of last modification of this filter. This is read-only field."
+  create_user_id:
+    type: integer
+    description: "ID of user that created this filter.  This is a read-only field."
+  modify_user_id:
+    type: integer
+    description: "ID of user that last modified this filter.  This is a read-only field."
+  version:
+    type: integer
+    description: "Optional version of the filter (not used yet)"
+  priority:
+    type: integer
+    description: "Optional - Priority of the filter. Higher priority filters are guaranteed to be used first. Negative priorities can be used as well. Default priority is 0"
+  internal:
+    type: boolean
+    description: "Internally provided exclusion (ex. through alliance). Not yet implemented."
+  enabled:
+    type: boolean
+    description: "Optional - True enables the filter (defaults to True)"
+  filter_on_sensor:
+    type: boolean
+    description: "True to push this filter to sensor. Note that, if filter was was pushed to sensor, it will not be enforced on the server. Note that only certain filters can be pushed to sensors today without loss of functionality. Defaults to False."
+  filter_on_server:
+    type: boolean
+    description: "True to filter on server. Defaults to true. If false, server will treat this filter as pass-through (disabled)."
+  track_metrics:
+    type: boolean
+    description: "Optional - True to capture metrics for this filter even when disabled. Defaults to true. Note: If we are metering disabled filter, we will resort to metering small sample set to reduce performance impact"
+  hit_rate:
+    type: float
+    description: "measured 15-min hit rate for a given filter as a rate (e.g. 0.01 equals to 1% hit rate). Note that this metrics will be available only for filters that are enabled or have trackMetrics turned on. Also, this metrics counts hit rate of the process, based on pathFilters, commandLineFilters and md5Filters. It doesn't take into the account the individual event hit rate."
+  name:
+    type: string
+    description: "Name of this filter - displayed in the UI"
```

Comment:
We don't have UI for this feature yet, so comment "displayed in the UI" should be removed from both this and the next property